databases/week2/queryPart_2sql

```diff
@@ -0,0 +1,27 @@
+-- Part 2 : School database
+CREATE DATABASE School;
+USE School;
+-- Create a new database containing the following tables: Class: with the columns: id, name, begins(date), ends(date)
+CREATE TABLE IF NOT EXISTS Class(
+    `id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(255) NOT NULL,
+    `begins_date` DATETIME NOT NULL,
+    `ends_date` DATETIME NOT NULL,
```

Comment:
why is the data type `Datetime` instead of `date` https://www.quora.com/SQL-Server-What-is-the-difference-between-datetime-and-date
_-- Create a new database containing the following tables: Class: with the columns: id, name, **begins(date), ends(date)**_

Reply:
i think i used autocompleted but then thought maybe it is good to indicate shows time as well